compiler/src/main/kotlin/walkers/visitor.kt

```diff
@@ -337,24 +370,43 @@ abstract class BaseASTVisitor: ASTVisitor {
     }
 
     override fun visit(node: PreIncStmt) {
-        // no-op
+        visit(node.variable)
```

Comment:
Remember that all pre/post, inc/dec statements have been removed. These are not present in neither the grammar nor compiler source on master.